App/App_macOS.swift

```diff
@@ -13,8 +13,15 @@ import Defaults
 import CoreKiwix
 
 #if os(macOS)
```

Comment:
As it turned out this is the official way of closing the application when the last window is closed.